frontend/map.component.ts

```diff
@@ -1,39 +1,65 @@
 import { Component, OnInit } from "@angular/core";
 
-import { Tweet } from "./tweet";
 import { MapService } from "./map.service";
+import { MapObject } from "./mapObject";
 
 @Component({
     selector: "map-app",
     templateUrl: "map.component.html",
     providers: [MapService]
 })
 export class MapComponent implements OnInit {
-    tweets: Tweet[];
+
+    map: google.maps.Map;
+    currentlyOpenInfoWindow: google.maps.InfoWindow = undefined;
```

Comment:
ist das `= undefined` nicht implizit mit dabei? in der zeile 13 wird es ja auch nicht verwendet. welchen weg wir verwenden ist mir egal. es sollte nur einheitlich sein.